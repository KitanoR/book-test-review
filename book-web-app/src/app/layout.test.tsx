import { render, screen } from "@testing-library/react";  
import RootLayout from "@/app/layout";

describe("RootLayout", () => {
  it("should render the layout", () => {
    const { asFragment } = render(<RootLayout>Test</RootLayout>);

    expect(screen.getByText("Test")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});