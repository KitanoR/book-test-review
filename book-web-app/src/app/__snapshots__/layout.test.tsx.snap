// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootLayout should render the layout 1`] = `<DocumentFragment />`;
